qbft/spectest/tests/commit/no_commit_quorum.go

```diff
@@ -0,0 +1,71 @@
+package commit
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	qbftcomparable "github.com/bloxapp/ssv-spec/qbft/spectest/comparable"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// NoCommitQuorum tests the state of the QBFT instance when received commit messages don't create a quorum
+func NoCommitQuorum() tests.SpecTest {
+	pre := testingutils.BaseInstance()
+	ks := testingutils.Testing4SharesSet()
+	sc := NoCommitQuorumStateComparison()
+
+	msgs := []*qbft.SignedMessage{
+		testingutils.TestingProposalMessage(ks.Shares[1], 1),
+
+		testingutils.TestingPrepareMessage(ks.Shares[1], 1),
+		testingutils.TestingPrepareMessage(ks.Shares[2], 2),
+		testingutils.TestingPrepareMessage(ks.Shares[3], 3),
+
+		testingutils.TestingCommitMessage(ks.Shares[1], 1),
+		testingutils.TestingCommitMessage(ks.Shares[2], 2),
+	}
+	return &tests.MsgProcessingSpecTest{
+		Name:          "no commit quorum",
+		Pre:           pre,
+		PostRoot:      sc.Root(),
+		PostState:     sc.ExpectedState,
+		InputMessages: msgs,
+		OutputMessages: []*qbft.SignedMessage{
+			testingutils.TestingPrepareMessage(ks.Shares[1], 1),
+			testingutils.TestingCommitMessage(ks.Shares[1], 1),
+		},
+	}
+}
+
+func NoCommitQuorumStateComparison() *qbftcomparable.StateComparison {
+	ks := testingutils.Testing4SharesSet()
+
+	state := testingutils.BaseInstance().State
+	state.ProposalAcceptedForCurrentRound = testingutils.TestingProposalMessage(ks.Shares[1], types.OperatorID(1))
+
+	state.LastPreparedRound = 1
+	state.LastPreparedValue = testingutils.TestingQBFTFullData
+
```

Comment:
specify here, even though by default, that decided=false

Reply:
done

Reply:
where?

Reply:
https://github.com/bloxapp/ssv-spec/pull/216/files#diff-0e78fc58e6c29d591e645957f5f351fb7b63c695b917bdc4026d2fe8f8b34d6fR48